scripts/devenv-builder/create-vm.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# This script automates the VM creation steps described in the "MicroShift Development Environment on RHEL 8" document.
+# See https://github.com/ggiguash/microshift/blob/main/docs/devenv_rhel8.md#creating-vm
+#
+set -eo pipefail
+ROOTDIR=$(git rev-parse --show-toplevel)/scripts/devenv-builder
+
+function usage() {
+    echo "Usage: $(basename $0) <vm_name> <rhel_iso_file> <ncpus> <memory_in_GB> <disk_in_GB> <data_vol_size_in_GB>" 
+    exit 1
+}
+
+if [ $# -ne 6 ] ; then
+    usage
+fi
+
+VMNAME=$1
+RHEL_ISO=$2
+NCPUS=$3
+RAMSIZE=$4
+DISKSIZE=$5
+DATAVOLSIZE=$6
+[ -z ${VMNAME} ] && usage
+[ ! -e "${RHEL_ISO}" ] && usage
+[ -z ${NCPUS} ] && usage
+[ -z ${RAMSIZE} ] && usage
+[ -z ${DISKSIZE} ] && usage
+[ -z ${DATAVOLSIZE} ] && usage
+
+# RAM size is expected in MB
+RAMSIZE=$(( ${RAMSIZE} * 1024 ))
+# Calculate system root partition size (1GB is allocated to the boot partition)
+SYSROOTSIZE=$(( ${DISKSIZE} - 1 - ${DATAVOLSIZE} ))
+# System root size is expected in MB
+SYSROOTSIZE=$(( ${SYSROOTSIZE} * 1024 ))
+
+KICKSTART_FILE=/tmp/devenv-kickstart.ks
+cat ${ROOTDIR}/config/kickstart.ks.template | \
+    sed "s;REPLACE_HOST_NAME;${VMNAME};" | \
+    sed "s;REPLACE_LVM_SYSROOT_SIZE;${SYSROOTSIZE};" > ${KICKSTART_FILE}
+
+sudo -b bash -c " \
+cd /var/lib/libvirt/images/ && \
+virt-install \
```

Comment:
I tried this and got an error the the volume pool would be exhausted by the space I asked to be given to the VM, so I think we're going to need some more control over where the VM is being created.